x-pack/plugins/endpoint/public/embeddables/resolver/store/data/selectors.ts

```diff
@@ -0,0 +1,302 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { createSelector } from 'reselect';
+import { ResolverState } from '../../types';
+import { sampleData } from './sample';
+import { depthFirstPreorder, levelOrder } from '../../lib/tree_sequencers';
+import { Vector2 } from '../../types';
+import { add as vector2Add } from '../../lib/vector2';
+
+const unit = 100;
+const distanceBetweenNodesInUnits = 1;
+const distanceBetweenNodes = distanceBetweenNodesInUnits * unit;
```

Comment:
:question: Have you tested making this _more_ than 1? Some ideas Lindsay and I are working on use space from a node's "descending" edge-line to lay out information. Would be super cool if this works (I assume it does).